src/main/components/Button.jsx

```diff
@@ -1,20 +1,36 @@
 import styled from 'styled-components';
 
+let backgroundColor;
+
 const Button = styled.button`
```

Comment:
This maybe for a new issue but the button appears to be fixed and does not adjust with the window size. When launching on my end,  the button span other panels.